sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -26,57 +25,40 @@
     KeyValue,
     AuthorizationRule as InternalAuthorizationRule,
 )
-
 from ._model_workaround import adjust_attribute_map, avoid_timedelta_overflow
 from ._constants import RULE_SQL_COMPATIBILITY_LEVEL
 from ._utils import _normalize_entity_path_to_full_path_if_needed
 
-adjust_attribute_map()
-
-
-# These helpers are to ensure that the Properties objects can't be constructed without all args present,
-# as a compromise between our use of kwargs to flatten arg-lists and trying to de-incentivise manual instantiation
-# while still trying to provide some guardrails.
-def extract_kwarg_template(kwargs, extraction_missing_args, name): # pylint: disable=inconsistent-return-statements
-    try:
-        return kwargs[name]
-    except KeyError:
-        extraction_missing_args.append(name)
+if TYPE_CHECKING:
+    from ._generated.models import (
+        MessagingSku,
+        NamespaceType,
+        EntityAvailabilityStatus,
+        EntityStatus,
+        AccessRights,
+        MessageCountDetails,
+    )
 
-
-def validate_extraction_missing_args(extraction_missing_args):
```

Comment:
double checking, we don't need this anymore?

Reply:
no, not after expanding the kwargs